wfe/web-front-end.go

```diff
@@ -312,7 +312,7 @@ func (wfe *WebFrontEndImpl) verifyPOST(request *http.Request, regCheck bool) ([]
 		return nil, nil, reg, errors.New("Too many signatures on POST")
 	}
 	if len(parsedJws.Signatures) == 0 {
-		wfe.log.Debug(fmt.Sprintf("POST not signed: %v", parsedJws))
+		wfe.log.Debug(fmt.Sprintf("POST not signed: %s", bodyStr))
```

Comment:
Very sorry to waste your time. I totally had fixed this.  No idea where that commit went.

Feel free to not review my code until the tests pass on it. It is ridic to expect you to waste time on knucklehead reviews.